Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp

```diff
@@ -2181,6 +2182,18 @@ void NetworkResourceLoader::useRedirectionForCurrentNavigation(WebCore::Resource
     m_redirectionForCurrentNavigation = makeUnique<WebCore::ResourceResponse>(WTFMove(response));
 }
 
+uint64_t NetworkResourceLoader::bytesTransferredOverNetwork()
```

Comment:
This name feels misleading, since it is really "total bytes transferred over the network since the last time you called this function", rather than "total bytes transferred over the network".

I think I would either rename this method to reflect that (`bytesTransferredOverNetworkDelta?`) or refactor to make this clearer.